### PR TITLE
Update setup documentation. Add macOS related commands.

### PR DESCRIPTION
**Pull request by eugenwick**
Had a few issues without these commands on my macOS High Sierra Setup.

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -345,3 +357,6 @@ gopass --yes setup --remote github.com/example/pass.git --alias example --create
 # For every other team member initialize a new store and clone the existing remote
 gopass --yes setup --remote github.com/example/pass.git --alias example --name "Jane Doe" --email "jane.doe@example.com"
 ```
+
```
Comment (metalmatze) [inline docs/setup.md]: Can you remove these 3 too many lines.

Comment (eugenwick) [reply docs/setup.md]: done.

Comment (metalmatze) [reply docs/setup.md]: Did you push your changes? It's still the old one.

Comment (eugenwick) [reply docs/setup.md]: Oops, sorry. Didn't add the changes before --amend :)

**Pull request merged**